src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+public class Lottery {
+    private static final int MAX_POSSIBLE_NUMBER = 100;
```

Comment:
Please avoid using static methods or fields unless they are constants. In this case, MAX_POSSIBLE_NUMBER is correctly defined as a constant.